time timestamp
string motor_type
int8  can_id
float32 raw_rpm
float32 current
float32 duty_cycle